packages/contracts-bedrock/src/universal/InitializablePublic.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// Interfaces
+import { Initializable } from "src/vendor/Initializable-v5.sol";
+
+/// @title InitializablePublic
+/// @notice Expands Initializable v5 vendored from OpenZeppelin to expose the version getter
+contract InitializablePublic is Initializable {
```

Comment:
I would suggest keeping this called `Initializeable` as it helps minimize the diff and is a more standard name, and we can just alias the import like so
```suggestion
// Interfaces
import { Initializable as InitializableOZ } from "src/vendor/Initializable-v5.sol";

/// @title Initializable
/// @notice Expands Initializable v5 vendored from OpenZeppelin to expose the version getter
contract Initializable is InitializableOZ {
```

Reply:
It looks fine, let's get @maurelian opinion

Reply:
I talked with him, I will do the changes